tests/container_test.py

```diff
@@ -20,7 +20,6 @@
 
 def test_container_count(dockerc):
     """Verify the test composition and container."""
-    # stopped parameter allows non-running containers in results
```

Comment:
Actually, maybe this comment should stay and we should just change the word `stopped` to `all`.  What do you think?

Reply:
I honestly don't care.  What do you prefer?

Reply:
Modifying the comment (as opposed to deleting it) preserves the original intention of explaining why we are passing in `all=True` to `dockerc.compose.ps`, so I prefer keeping it.

Reply:
Please see commit a518cfb.

Reply:
Thanks!